src/schema/schema_create.c

```diff
@@ -433,6 +433,57 @@ __wt_find_import_metadata(WT_SESSION_IMPL *session, const char *uri, const char
     return (0);
 }
 
+/*
+ * __schema_is_tiered_storage_shared --
+ *     Check whether the table is configured for a tiered storage shared.
```

Comment:
Do you mean:
 Check whether the table is configured for tiered storage, and if so, whether the tiered table is shared.

Reply:
Yes, fix it using the above-suggested statement.